Vagrantfile

```diff
@@ -7,6 +7,7 @@ Vagrant.configure(2) do |config|
         d.has_ssh = true
```

Comment:
On the 3rd line, i believe this should be added: 
` ENV['VAGRANT_DEFAULT_PROVIDER'] = 'docker' `
Tested qemu tests with this and the taps are being successfully created.